submission.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Virametric</title>
```

Comment:
I'll suggest you add more contact on the name of the page
```suggestion
        <title>Virametric - News Submission</title>
```